src/components/Section.js

```diff
@@ -169,7 +172,7 @@ export class Section extends React.Component {
       {titleDiv}
       <div className="section-content">
         {imageList}
-        {section.type !== "main" ?
+        {section.type !== "main" && section.abstract ?
```

Comment:
The problem here is that section.abstract is always present even if empty (an empty object evaluates as true due to ES6 silliness). `getAttr(section.abstract, language)` will return `undefined` if the object is empty, that should be good :D